frontend/integration-tests/protractor.conf.ts

```diff
@@ -113,8 +121,8 @@ export const config: Config = {
       'tests/source-to-image.scenario.ts',
       'tests/deploy-image.scenario.ts',
     ]),
-    // TODO(alecmerdler): Rename this to `olm` (being used by ci-operator)
     olmFull: suite([
+      'tests/olm/crd.scenario.ts',
```

Comment:
We should give this a more specific name if it's specific to OLM and not a general CRD test scenario.